packages/core/bootstrap/src/lib/util.ts

```diff
@@ -470,3 +471,22 @@ export const buildUrl = (baseUrl: string, pathTemplate = '', params = {}, whitel
   new URL(buildUrlPath(pathTemplate, params, whitelist), baseUrl).toString()
 
 //  URL Encoding
+
+let unhandledRejectionHandlerRegistered = false
+/**
+ * Adapters use to run with Node 14, which by default didn't crash when a rejected promised bubbled up to the top.
+ * This function registers a global handler to catch those rejections and just log them to console.
+ */
+export function registerUnhandledRejectionHandler(): void {
```

Comment:
Nitpick, but just for style consistency

```suggestion
let unhandledRejectionHandlerRegistered = false

/**
 * Adapters use to run with Node 14, which by default didn't crash when a rejected promised bubbled up to the top.
 * This function registers a global handler to catch those rejections and just log them to console.
 */
export const registerUnhandledRejectionHandler = (): void => {
```